Change the absolute paths included in the man pages so that the correct
values can be dropped in by config.status.  After applying this patch,
these files should be renamed to their ".in" counterparts, and then the
configure script should be rebuilt.
    appl/bsd/klogind.M
    appl/bsd/kshd.M
    appl/bsd/login.M
    appl/bsd/rcp.M
    appl/bsd/rlogin.M
    appl/bsd/rsh.M
    appl/bsd/v4rcp.M
    appl/gssftp/ftpd/ftpd.M
    appl/gssftp/ftp/ftp.M
    appl/sample/sclient/sclient.M
    appl/sample/sserver/sserver.M
    appl/telnet/telnetd/telnetd.8
    appl/telnet/telnet/telnet.1
    clients/kcpytkt/kcpytkt.M
    clients/kdeltkt/kdeltkt.M
    clients/kdestroy/kdestroy.M
    clients/kinit/kinit.M
    clients/klist/klist.M
    clients/kpasswd/kpasswd.M
    clients/ksu/ksu.M
    clients/kvno/kvno.M
    config-files/kdc.conf.M
    config-files/krb5.conf.M
    gen-manpages/k5login.M
    gen-manpages/kerberos.M
    kadmin/cli/k5srvutil.M
    kadmin/cli/kadmin.local.M
    kadmin/cli/kadmin.M
    kadmin/dbutil/kdb5_util.M
    kadmin/ktutil/ktutil.M
    kadmin/passwd/kpasswd.M
    kadmin/server/kadmind.M
    kdc/fakeka.M
    kdc/krb5kdc.M
    krb524/k524init.M
    krb524/krb524d.M
    krb5-config.M
    plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M
    slave/kpropd.M
    slave/kprop.M
    tests/create/kdb5_mkdums.M
    util/et/com_err.3
    util/et/compile_et.1
    util/profile/profile.5
    util/send-pr/send-pr.1
Index: configure.in
===================================================================
--- configure.in	(revision 19589)
+++ configure.in	(working copy)
@@ -986,6 +986,73 @@
 KRB5_WITH_PAM
 
 AC_CONFIG_FILES(krb5-config, [chmod +x krb5-config])
+
+mansysconfdir=$sysconfdir
+mansysconfdir=`eval echo $mansysconfdir | sed -e "s,NONE,$prefix,g"`
+mansysconfdir=`eval echo $mansysconfdir | sed -e "s,NONE,$ac_default_prefix,g"`
+mansbindir=$sbindir
+mansbindir=`eval echo $mansbindir | sed -e "s,NONE,$exec_prefix,g"`
+mansbindir=`eval echo $mansbindir | sed -e "s,NONE,$prefix,g"`
+mansbindir=`eval echo $mansbindir | sed -e "s,NONE,$ac_default_prefix,g"`
+manlocalstatedir=$localstatedir
+manlocalstatedir=`eval echo $manlocalstatedir | sed -e "s,NONE,$prefix,g"`
+manlocalstatedir=`eval echo $manlocalstatedir | sed -e "s,NONE,$ac_default_prefix,g"`
+manlibexecdir=$libexecdir
+manlibexecdir=`eval echo $manlibexecdir | sed -e "s,NONE,$exec_prefix,g"`
+manlibexecdir=`eval echo $manlibexecdir | sed -e "s,NONE,$prefix,g"`
+manlibexecdir=`eval echo $manlibexecdir | sed -e "s,NONE,$ac_default_prefix,g"`
+AC_SUBST(mansysconfdir)
+AC_SUBST(mansbindir)
+AC_SUBST(manlocalstatedir)
+AC_SUBST(manlibexecdir)
+AC_OUTPUT([
+	appl/bsd/klogind.M
+	appl/bsd/kshd.M
+	appl/bsd/login.M
+	appl/bsd/rcp.M
+	appl/bsd/rlogin.M
+	appl/bsd/rsh.M
+	appl/bsd/v4rcp.M
+	appl/gssftp/ftpd/ftpd.M
+	appl/gssftp/ftp/ftp.M
+	appl/sample/sclient/sclient.M
+	appl/sample/sserver/sserver.M
+	appl/telnet/telnetd/telnetd.8
+	appl/telnet/telnet/telnet.1
+	clients/kcpytkt/kcpytkt.M
+	clients/kdeltkt/kdeltkt.M
+	clients/kdestroy/kdestroy.M
+	clients/kinit/kinit.M
+	clients/klist/klist.M
+	clients/kpasswd/kpasswd.M
+	clients/ksu/ksu.M
+	clients/kvno/kvno.M
+	config-files/kdc.conf.M
+	config-files/krb5.conf.M
+	gen-manpages/k5login.M
+	gen-manpages/kerberos.M
+	kadmin/cli/k5srvutil.M
+	kadmin/cli/kadmin.local.M
+	kadmin/cli/kadmin.M
+	kadmin/dbutil/kdb5_util.M
+	kadmin/ktutil/ktutil.M
+	kadmin/passwd/kpasswd.M
+	kadmin/server/kadmind.M
+	kdc/fakeka.M
+	kdc/krb5kdc.M
+	krb524/k524init.M
+	krb524/krb524d.M
+	krb5-config.M
+	plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M
+	slave/kpropd.M
+	slave/kprop.M
+	tests/create/kdb5_mkdums.M
+	util/et/com_err.3
+	util/et/compile_et.1
+	util/profile/profile.5
+	util/send-pr/send-pr.1
+])
+
 V5_AC_OUTPUT_MAKEFILE(.
 
 	util util/support util/profile util/send-pr
Index: appl/sample/sserver/sserver.M
===================================================================
--- appl/sample/sserver/sserver.M	(revision 19589)
+++ appl/sample/sserver/sserver.M	(working copy)
@@ -59,7 +59,7 @@
 using a line in
 /etc/inetd.conf that looks like this:
 .PP
-sample  stream  tcp     nowait  root /usr/local/sbin/sserver	sserver
+sample  stream  tcp     nowait  root @mansbindir@/sserver	sserver
 .PP
 Since \fBsample\fP is normally not a port defined in /etc/services, you will
 usually have to add a line to /etc/services which looks like this:
Index: appl/bsd/klogind.M
===================================================================
--- appl/bsd/klogind.M	(revision 19589)
+++ appl/bsd/klogind.M	(working copy)
@@ -27,7 +27,7 @@
 the port indicated in /etc/inetd.conf.  A typical /etc/inetd.conf
 configuration line for \fIklogind\fP might be:
 
-klogin stream tcp nowait root /usr/cygnus/sbin/klogind klogind -e5c
+klogin stream tcp nowait root @mansbindir@/klogind klogind -e5c
 
 When a service request is received, the following protocol is initiated:
 
Index: appl/bsd/kshd.M
===================================================================
--- appl/bsd/kshd.M	(revision 19589)
+++ appl/bsd/kshd.M	(working copy)
@@ -8,7 +8,7 @@
 .SH NAME
 kshd \- kerberized remote shell server
 .SH SYNOPSIS
-.B /usr/local/sbin/kshd 
+.B @mansbindir@/kshd 
 [
 .B \-kr45ec
 ]
@@ -30,7 +30,7 @@
 on the port indicated in /etc/inetd.conf.  A typical /etc/inetd.conf
 configuration line for \fIkrshd\fP might be:
 
-kshell	stream	tcp	nowait	root	/usr/local/sbin/kshd	kshd -5c
+kshell	stream	tcp	nowait	root	@mansbindir@/kshd	kshd -5c
 
 When a service request is received, the following protocol is initiated:
 
Index: appl/telnet/telnetd/telnetd.8
===================================================================
--- appl/telnet/telnetd/telnetd.8	(revision 19589)
+++ appl/telnet/telnetd/telnetd.8	(working copy)
@@ -37,7 +37,7 @@
 .SM DARPA TELNET
 protocol server
 .SH SYNOPSIS
-.B /usr/libexec/telnetd
+.B @manlibexecdir@/telnetd
 [\fB\-a\fP \fIauthmode\fP] [\fB\-B\fP] [\fB\-D\fP] [\fIdebugmode\fP]
 [\fB\-e\fP] [\fB\-h\fP] [\fB\-I\fP\fIinitid\fP] [\fB\-l\fP]
 [\fB\-k\fP] [\fB\-n\fP] [\fB\-r\fP\fIlowpty-highpty\fP] [\fB\-s\fP]
Index: kadmin/cli/kadmin.M
===================================================================
--- kadmin/cli/kadmin.M	(revision 19589)
+++ kadmin/cli/kadmin.M	(working copy)
@@ -826,9 +826,9 @@
 .RS
 .TP
 EXAMPLE:
-kadmin: ktremove -k /usr/local/var/krb5kdc/kadmind.keytab kadmin/admin
+kadmin: ktremove -k @manlocalstatedir@/krb5kdc/kadmind.keytab kadmin/admin
 Entry for principal kadmin/admin with kvno 3 removed
-	from keytab WRFILE:/usr/local/var/krb5kdc/kadmind.keytab.
+	from keytab WRFILE:@manlocalstatedir@/krb5kdc/kadmind.keytab.
 kadmin:
 .RE
 .fi
@@ -870,7 +870,7 @@
 .SH HISTORY
 The
 .B kadmin
-prorgam was originally written by Tom Yu at MIT, as an interface to the
+program was originally written by Tom Yu at MIT, as an interface to the
 OpenVision Kerberos administration program.
 .SH SEE ALSO
 .IR kerberos (1),
Index: config-files/kdc.conf.M
===================================================================
--- config-files/kdc.conf.M	(revision 19589)
+++ config-files/kdc.conf.M	(working copy)
@@ -82,14 +82,14 @@
 .B string
 specifies the location of the access control list (acl) file that
 kadmin uses to determine which principals are allowed which permissions
-on the database. The default value is /usr/local/var/krb5kdc/kadm5.acl.
+on the database. The default value is @manlocalstatedir@/krb5kdc/kadm5.acl.
 
 .IP admin_keytab
 This
 .B string
 Specifies the location of the keytab file that kadmin uses to
 authenticate to the database.  The default value is
-/usr/local/var/krb5kdc/kadm5.keytab.
+@manlocalstatedir@/krb5kdc/kadm5.keytab.
 
 .IP database_name
 This
@@ -239,7 +239,7 @@
 realm names and the [capaths] section of its krb5.conf file
 
 .SH FILES 
-/usr/local/var/krb5kdc/kdc.conf
+@manlocalstatedir@/krb5kdc/kdc.conf
 
 .SH SEE ALSO
 krb5.conf(5), krb5kdc(8)
Index: slave/kprop.M
===================================================================
--- slave/kprop.M	(revision 19589)
+++ slave/kprop.M	(working copy)
@@ -39,7 +39,7 @@
 This is done by transmitting the dumped database file to the slave
 server over an encrypted, secure channel.  The dump file must be created
 by kdb5_util, and is normally KPROP_DEFAULT_FILE
-(/usr/local/var/krb5kdc/slave_datatrans).
+(@manlocalstatedir@/krb5kdc/slave_datatrans).
 .SH OPTIONS
 .TP
 \fB\-r\fP \fIrealm\fP
@@ -51,7 +51,7 @@
 \fB\-f\fP \fIfile\fP
 specifies the filename where the dumped principal database file is to be
 found; by default the dumped database file is KPROP_DEFAULT_FILE
-(normally /usr/local/var/krb5kdc/slave_datatrans).
+(normally @manlocalstatedir@/krb5kdc/slave_datatrans).
 .TP
 \fB\-P\fP \fIport\fP
 specifies the port to use to contact the
Index: slave/kpropd.M
===================================================================
--- slave/kpropd.M	(revision 19589)
+++ slave/kpropd.M	(working copy)
@@ -69,7 +69,7 @@
 This is done by adding a line to the inetd.conf file which looks like
 this:
 
-kprop	stream	tcp	nowait	root	/usr/local/sbin/kpropd	kpropd
+kprop	stream	tcp	nowait	root	@mansbindir@/kpropd	kpropd
 
 However, kpropd can also run as a standalone deamon, if the
 .B \-S
@@ -87,13 +87,13 @@
 \fB\-f\fP \fIfile\fP
 specifies the filename where the dumped principal database file is to be
 stored; by default the dumped database file is KPROPD_DEFAULT_FILE
-(normally /usr/local/var/krb5kdc/from_master).
+(normally @manlocalstatedir@/krb5kdc/from_master).
 .TP
 .B \-p
 allows the user to specify the pathname to the
 .IR kdb5_util (8)
 program; by default the pathname used is KPROPD_DEFAULT_KDB5_UTIL
-(normally /usr/local/sbin/kdb5_util).
+(normally @mansbindir@/kdb5_util).
 .TP
 .B \-S
 turn on standalone mode.  Normally, kpropd is invoked out of
@@ -124,14 +124,14 @@
 allows the user to specify the path to the
 .KR kpropd.acl
 file; by default the path used is KPROPD_ACL_FILE
-(normally /usr/local/var/krb5kdc/kpropd.acl).
+(normally @manlocalstatedir@/krb5kdc/kpropd.acl).
 .SH FILES
 .TP "\w'kpropd.acl\ \ 'u"
 kpropd.acl
 Access file for
 .BR kpropd ;
 the default location is KPROPD_ACL_FILE (normally
-/usr/local/var/krb5kdc/kpropd.acl).
+@manlocalstatedir@/krb5kdc/kpropd.acl).
 Each entry is a line containing the principal of a host from which the
 local machine will allow Kerberos database propagation via kprop.
 .SH SEE ALSO
